opentreemap/treemap/js/src/lib/inlineEditForm.js

```diff
@@ -19,6 +19,7 @@ exports.init = function(options) {
         form = options.form || section + ' form',
         $section = $(section),
         edit = options.edit || '.editBtn',
+        $edit = $(edit),
```

Comment:
@RickMohr and @jwalgran, the only place where the edit button gets dynamically loaded appears to be [in mapFeatureDetail.js](https://github.com/OpenTreeMap/otm-core/blob/develop/opentreemap/treemap/js/src/mapFeatureDetail.js#L119), handling save success, which is irrelevant because if the user wasn't logged in, they would not have saved changes, never mind successfully.

Is it safe to assume that the base page is loaded before the javascript executes?

I assume that we do not care about any possible future case that could dynamically load details with the edit button during initial page load.